crates/daphne-server/src/roles/aggregator.rs

```diff
@@ -41,21 +44,10 @@ impl DapAggregator<DaphneAuth> for crate::App {
         let task_id_hex = task_id.to_hex();
         let durable = self.durable();
 
-        let skip_replay_protection = self
-            .kv()
-            .get_cloned::<kv::prefix::GlobalConfigOverride<bool>>(
-                &kv::prefix::GlobalOverrides::SkipReplayProtection,
-                &KvGetOptions {
-                    cache_not_found: true,
-                },
-            )
-            .await
-            .inspect_err(
-                |e| tracing::error!(error = ?e, "failed to fetch skip_replay_protection from kv"),
-            )
-            .ok() // treat error as false
-            .flatten()
-            .unwrap_or_default(); // treat missing as false
+        let skip_replay_protection = matches!(
```

Comment:
I see there's a helper for `enabled()`: https://github.com/cloudflare/daphne/pull/621/files#diff-7c6740c5b3ceb5af4a8f7daa7f7e1adec53ccf3dda8eb714c918fcd860697dbfR385

Should we make one for disabled too?